salt/kubelet/node_labels.jinja

```diff
@@ -0,0 +1 @@
+KUBELET_NODE_LABELS="--node-labels={{ node_labels|join(",") }}"
```

Comment:
I agree with @kiall on the solution: add an `-initial` file and just do not `watch` it. That would be enough.

Just one thing: I remember having problems when trying to concat envars like `KUBELET_ARGS="$KUBELET_ARGS ..."`, so maybe you have to use different names...